### PR TITLE
UG Bug Fixes

### PR DESCRIPTION
**Pull request by koh-jx**
# Notable UG changes

1. Notable prefix formatting section
2. GUI section

Not sure how to fix issue 159 and 160

# Bugs and stuff

- Due to the current bug of the tags shifting the button out of the dialog window when adding task via GUI, unable to take a screenshot as of now.
-  Also, pressing 'Enter' in the title will not save the task but rather bring the cursor to the description textbox. To create an issue for that later.

Currently assuming that no warning is given about invalid arguments in the GUI add dialog. If this bug is fixed, will need to change this.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -75,10 +75,24 @@ use via a CLI while still having the benefits of a GUI.
     - If a parameter is expected only once in the command, but you specified it multiple times,
       an error message will be shown.
       
-    - If an extraneous parameter is specified, an error message will be shown.
+    - If an extraneous parameter is specified, an error message will be shown. (Does not apply for tags or contacts)
+    
+    - Parameter or argument values cannot contain the character `/`. An error message will be shown if `/` is present in the argument value. For example, `task add 1/2` is invalid, and can be replaced with valid alternatives such as `task add 1 or 2` instead.
```
Comment (jeffsieu) [inline docs/UserGuide.md:80]: `task add 1` or `task add 2`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -75,10 +75,24 @@ use via a CLI while still having the benefits of a GUI.
...
+
+Some prefixes require the user to follow certain formats in their commands.
+
+| **Prefix**  | **Use** | **Format** |
+| ------------- | ------------- | ------------- |
+| `ts`  | Timestamp, represents a date  | `DD`-`MM`-`YYYY` <br/><br/> 1. Single-digit dates or months require a `0` appended at the front. For example, 3rd July 2021 is represented as `03-07-2021`. <br/><br/> 2. The day, month and year are separated by hyphens. |
+| `e`  | Email  | `local-part@domain` <br/><br/> 1. The local-part should only contain alphanumeric characters and these special characters, excluding the parentheses, (+_.-). The local-part may not start or end with any special characters.<br/><br/>2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods. <br/><br/> The domain name must: <br/>     - end with a domain label at least 2 characters long <br/> - have each domain label start and end with alphanumeric characters <br/> - have each domain label consist of alphanumeric characters, separated only by hyphens, if any. |
+
+
+<a name="ab3comm"></a>
```
Comment (jeffsieu) [inline docs/UserGuide.md:94]: What's this for?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -124,6 +136,8 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 - When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 - You can remove all the person’s tags by typing `t/` without
   specifying any tags after it.
+- If the edited Person's name corresponds to a name of a contact in any task, the corresponding tasks' contact name will be changed accordingly as well. (if applicable)
```
Comment (jeffsieu) [inline docs/UserGuide.md:139]: contact names

**Review by jeffsieu — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (koh-jx) [reply docs/UserGuide.md:80]: wait what no, this example is talking about if like people wanna have multiple choices in their task title, not about multiple tasks.

For eg) `task add Bake cake/cookies` will have to become `task add Bake cake or cookies`

File: docs/UserGuide.md (hunk 2 above)
Comment (koh-jx) [reply docs/UserGuide.md:94]: Email format provided by the AB3 itself. 
To fix #110 , think i forgot to link it

Comment (jeffsieu) [reply docs/UserGuide.md:94]: But what is this for?
`<a name="ab3comm"></a>`

Comment (koh-jx) [reply docs/UserGuide.md:94]: oh er I think its part of the UG by default? I didn't touch this.

The default one has stuff like
`<a name="features"></a>`
`<a name="commandsummary"></a>` etc
I'm not sure if it's needed but don't want to risk not grading or whatever.

File: docs/UserGuide.md (hunk 1 above)
Comment (jeffsieu) [reply docs/UserGuide.md:80]: Right.

**Review by jeffsieu — APPROVED** [review]

**Pull request merged**